[PROOFREADING] wallet/Electrum - Italian

Bitman-ops opened the pull request:
NOTE:

- Line 97 talks about bold but bold is not present in the it version or in the en version.
- Lines 671 and 731 it says "See here" meaning a link but the link is not present in the it version or in the en version.
- Lines 231-233-511 I left the word "monete" as is because Electrum uses this word in its Italian menu.

I converted all the remaining commands that refer to the images into English to be consistent with the images themselves that have English writing.

Bitman-ops committed:
Italian proofreading - Electrum

Bitman-ops committed:
Italian proofreading - Electrum

Bitman-ops committed:
Guida completa a Electrum, da zero a eroe

proofredato fino riga 206

Bitman-ops committed:
Electrum

Bitman-ops committed:
Electrum

MariJJhodl committed:
First check

MariJJhodl commented:
Hello @Bitman-ops I noticed you changed the title of Electrum sections in Italian, following the translation of the tool you probably have installed.
However, you didn't do the same with the commands, which where left in English, like finalize, "save", "combine", "sign" e "broadcast" etc.
So, here there is incoherence for the Italian user. At this point, since images are in English, my suggestion would be to leave everything in EN (even tab titles) and maybe add a parenthesis after tab titles, and after commands, saying something like (nella versione in italiano, "monete") or (nella versione in italiano, "finalizzare") according to what you have in your version.

Can you make these modifications?

MariJJhodl committed:
second big fix

Bitman-ops committed:
Italian proofreading - Electrum

287

Bitman-ops committed:
Merge branch 'PlanB-Network:dev' into patch-19

Bitman-ops committed:
Italian proofreading - Electrum

I revised the tutorial by inserting the command names first in English and then, in parentheses, in Italian.

Fixed typos.

Since I don't have funds on Electrum, I wasn't able to explore some parts/commands.

For example: on line 612, I don't know how the "combine" and "broadcast" buttons are translated into Italian. I also don't know what a "watching wallet" is called in Italian.

On lines 848 and 764, I left the commands in English only, as they're talking about Coldcard, which I don't have.

Bitman-ops committed:
Merge branch 'PlanB-Network:dev' into patch-19

Bitman-ops committed:
Italian proofreading - Electrum

Fixed watching wallet - portafogli di sola visione
